pkg/worker/worker.go

```diff
@@ -58,6 +58,15 @@ func (w *Worker) Work() {
 	mgr.Register("DigestEmailJob", w.DigestEmailJob)
 	mgr.Register("AggregatedDigestEmailJob", w.AggregatedDigestEmailJob)
 
+	/**********************
+	* //Future Enhancement
+	* Consider providing workers with dataloaders, and potentially a shared context. As these run separate go routines for each worker,
+	***********************
+	*dataLoaders := loaders.NewDataLoaders(w.Store)
+	*ctx := loaders.CTXWithLoaders(context.Background(), dataLoaders)
+	*err := mgr.RunWithContext(ctx)
+	******************************/
```

Comment:
This is a great suggestion, but I wonder if this is the best way to capture these future investments. I've seen many ancient future TODO's in code bases forgotten to the annals of time.

Reply:
Thanks, it's true we could make a tech debt ticket for this. However, there is not a large benefit at this point of time. (I started to implement this here, but it is too much scope creep.)

Making this note seemed to a good middle ground to me. This way if there is another ticket where this task could be in scope, there is a note in the codebase. If we need to expand our use of factory in the future, we might consider it more.